mtls-examples/go/mtls.go

```diff
@@ -11,7 +11,7 @@ import (
 
 var (
 	// cert file MUST be a concatenation of certificate provided by Lean AND the certificate chain!!!
-	certFile = flag.String("cert", "cert_and_cert_chain.crt", "Your application certificate provided by Lean concatenated with cert chain")
+	certFile = flag.String("cert", "cert.crt", "Your application certificate provided by Lean concatenated with cert chain")
```

Comment:
I think the previous one was working, one of our clients said so. Can we confirm?

Reply:
This is just a name change. 
Keeping it `cert_and_cert_chain.crt` might be misleading as with the change to mTLS certificate issuing from developer portal, we are now giving our clients the *cert AND the chain* in the single file `cert.crt`